yjit/src/codegen.rs

```diff
@@ -4021,6 +4021,113 @@ fn jit_rb_false(
     true
 }
 
+
+/// Codegen for Kernel::is_a?
```

Comment:
```suggestion
/// Codegen for Kernel#is_a?
```

It seems less confusing to use the common format (the one used in documentation) to refer to a method name: `Foo.bar` (class method), `Foo#bar` (instance method), or `Foo.#bar` (both).

Reply:
(well, I just duplicated the point made in https://github.com/ruby/ruby/pull/7297#discussion_r1105080019)